shapely/geometry/base.py

```diff
@@ -759,6 +759,35 @@ def segmentize(self, tolerance, **kwargs):
         """
         return shapely.segmentize(self, tolerance, **kwargs)
 
+    def reverse(self, **kwargs):
+        """Returns a copy of this Geometry with the order of coordinates reversed.
+
+        If the Geometry is a polygon with interior rings, the interior rings are also
+        reversed.
+
+        Points are unchanged.
+
+        Parameters
+        ----------
+        **kwargs
+            For other keyword-only arguments, see the
+            `NumPy ufunc docs <https://numpy.org/doc/stable/reference/ufuncs.html#ufuncs-kwargs>`_.
+
+        See also
+        --------
+        is_ccw : Checks if a Geometry is clockwise.
+
+        Examples
+        --------
+        >>> line = Geometry("LINESTRING (0 0, 1 2)")
```

Comment:
On the latest main branch, those examples have been updated to no longer use `Geometry(..)`, see

https://github.com/shapely/shapely/blob/fc611de55001d148ea87e751a08dbb253622ea8c/shapely/constructive.py#L688-L694